### ISSUE TITLE
checkbox both checked and disbled not possible in pico 2

### ISSUE THREAD

**Issue by ghost** [picocss/pico#391]
## Describe the issue

There's no option to make a disabled checkbox appear checked. This works in pico 1.

### Current Behavior

Shows empty checkbox that can't be toggled, even though it is `checked`.

### Expected Behavior

Disabled, checked checkbox should be both disabled and visible as being checked.

### Reproduction URL

```
<label><input type="checkbox" name="foo" disabled checked> Foo Label</label>
```

### Environment

Linux, Firefox, Pico 2.0, commit 8ba5740b8a1a38435fcf3b29cce631f379259751

**Comment by peteringram0** [comment]
This looks fine for me on OSX, Chrome and FF from your example above. Are you still seeing this issue?

![checked disbaled](https://github.com/picocss/pico/assets/6588942/61799b69-5b83-4e04-9084-fa2d550e4ba9)

![checked](https://github.com/picocss/pico/assets/6588942/9632f351-f208-4f6a-a5e9-00ef5aba75ae)

**Comment by drittich** [comment]
I've reproduced. Works in Chrome on Windows, does not work in FF on Windows. It appears in FF on Windows that the background color has a higher stacking order than background image.

![image](https://github.com/picocss/pico/assets/1222810/0be99131-5bfb-42da-a228-3add467eabd0)

If I remove the background color style it works:

![image](https://github.com/picocss/pico/assets/1222810/8171e1c4-ab30-4e04-8e93-6f04b5ecf37a)

**Comment by harristom** [comment]
Same issue on Edge 120.0.2210.144, Chrome 120.0.6099.225, and Firefox 122.0 on Windows 10

**Comment by lucaslarroche** [comment]
@ghost, thank you for spotting the bug.
@drittich, I cannot see the bug on macOS/iOS, but your screenshots are very helpful.

As an easy fix, I will remove the `--pico-form-element-disabled-background-color` and `--pico-form-element-disabled-border-color` CSS vars as they are not used in the default theme (same color).

It will be shipped in the release candidate 4 (`2.0.0-rc4`).

**Comment by lucaslarroche** [comment]
This is now fixed with the v2 release. I'm closing this issue.
